server/models/service-process.js

```diff
@@ -205,4 +205,120 @@ module.exports = function extendServiceProcess(ServiceProcess) {
     });
   }
   ServiceProcess.prototype._getActAndHost = _getActAndHost;
+
+  /**
+   * Start tracking objects on a worker.
+   *
+   * @param {function} callback Callback function.
+   */
+  function startObjectTracking(callback) {
+    var self = this;
+    self.serviceInstance(function(err, instance) {
```

Comment:
It might be worth surfacing a class/static variant of `ServiceInstance#_appCommand` to reduce the boilerplate in these 6 very similar methods.